test/test18.sh

```diff
@@ -0,0 +1,9 @@
+printf "build/marc --output=test.tmp input/test18.opl\n";
+
+export LD_LIBRARY_PATH=build/
+build/alex --output=test.tmp input/test18.opl
```

Comment:
Please save output to tmp/test18.lex. This way the Makefile target "clean" will get rid of the test output files.